content/overview/pricing/rates.md

```diff
@@ -15,44 +15,42 @@ cloud.gov fees are broken into three components:
 
-Agencies pay through [an Interagency Agreement (IAA) with 18F]({{< relref "overview/overview/how-to-purchase.md" >}}).
+Agencies pay through [an Interagency Agreement (IAA) with 18F]({{< relref "overview/pricing/how-to-purchase.md" >}}).
 
 ## Access package
 
-You select a package based on the kinds of applications to be hosted. [Find out more about what's included in all access packages.]({{< relref "overview/pricing/whats-included.md" >}})
+You select a package based on the kinds of applications you plan to host. The access package sets the base price and limits on usage for your cloud.gov account. [Find out more about what’s included in all access packages.]({{< relref "overview/pricing/whats-included.md" >}})
 
-| Package | What's included? | Base annual fee\* | Application resource usage | Managed services available |
+| Package | What’s included? | Base annual fee\* | Resource usage quota | Managed services available |
 | --- | --- | --- | --- | --- |
-| **Sandbox** | Any user with a `.gov` or `.mil` email address can try out cloud.gov, no paperwork required. **Application resource usage is capped at 1GB.** | Free | Free | Only free services |
+| **Sandbox** | Any user with a `.gov` or `.mil` email address can try out cloud.gov for a limited time with no paperwork required. **Application resource usage is capped at 1GB.** | Free | Free | Only free services |
 | **Prototyping** | Suitable for many teams to deploy apps, though limited to the `*.app.cloud.gov` domain. Access control can be delegated to teams. No production data allowed. Usually purchased per agency/department. | $15K |  ~$99/GB/month | All\** |
 | **Open Data** | One public-facing Open Data (no confidentiality risk assessed) system, including all the spaces needed and DNS support. **This account is limited to 2GB of memory usage per month.** | $10K | ~$99/GB/month | All\** (up to $2500/year) |
 | **FISMA Low** | One public-facing FISMA Low system, including all the spaces needed and DNS support. | $20K | ~$99/GB/month | All\** |
 | **FISMA Moderate** | One public-facing FISMA Moderate system, including all the spaces needed and DNS support. Additional support for FISMA Moderate data requirements. | $90K | ~$99/GB/month | All\** |
 
 \*Access agreements are severable, and the access package fees are invoiced monthly.
 
-\*\*We aren't yet charging for paid services. See [managed services](#managed-services) for details.
+\*\*We aren’t yet charging for paid services. See [managed services](#managed-services) for details.
 
 ## Resource usage quota
 
-You will be charged for the **quota** you set for your resource usage. This is only based on the memory, in MB of RAM, your reserve for your applications.
+Along with the access package you select, you will be charged for the **quota** you set for your resource usage. The quota is the memory, in MB of RAM, your reserve for your applications. Unused quotas are not refunded, and they do not ‘roll over’ month to month.
```

Comment:
This might be more clear as "you reserve for your applications" and "Unused memory is not refunded"